Yosemite/YosemiteTests/Model/CardPresentConfigurationTests.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import XCTest
+@testable import Yosemite
+
+class CardPresentConfigurationTests: XCTestCase {
+    // MARK: - US Tests
+    func testConfigurationForUsWithStripeEnabledCanadaEnabled() throws {
```

Comment:
Would you mind updating the test names to match the preferred style of underscore-separated words? p91TBi-35M-p2 (there may be a better link!)